addons/account_qr_code_sepa/models/res_bank.py

```diff
@@ -27,6 +28,24 @@ def _get_qr_code_url(self, qr_method, amount, currency, debtor_partner, free_com
                 comment[:141],                                          # Remittance Information (Unstructured) (can't be set if there is a structured one)
                 '',                                                     # Beneficiary to Originator Information
             ]
+            return qr_code_vals
+        return super()._get_qr_vals(qr_method, amount, currency, debtor_partner, free_communication, structured_communication)
+
+    def _get_qr_code_base64(self, qr_method, amount, currency, debtor_partner, free_communication, structured_communication):
+        if qr_method == 'sct_qr':
+            qr_code_vals = self.get_qr_vals(amount, currency, debtor_partner, free_communication, structured_communication)
```

Comment:
Aren't you forgetting to pass the QR method here ? (Did you test ? You always should)

Reply:
Agreed. I wanted to do like _get_qr_code_url who is override in account_qr_code_sepa.
In this case, do you think I should also move the override of _get_qr_code_url to account module? (or maybe not possible in stable...?)